docs/apache-airflow/installation/supported-versions.rst

```diff
@@ -60,8 +60,8 @@ They are based on the official release schedule of Python and Kubernetes, nicely
 
 2. The "oldest" supported version of Python/Kubernetes is the default one. "Default" is only meaningful
    in terms of "smoke tests" in CI PRs which are run using this default version and default reference
-   image available in DockerHub. Currently the ``apache/airflow:latest`` and ``apache/airflow:2.5.2`` images
-   are Python 3.8 images, however, in the first MINOR/MAJOR release of Airflow released after 14.09.2023,
+   image available in DockerHub. Currently the ``apache/airflow:latest`` and ``apache/airflow:2.10.2`` images
+   are Python 3.8 images, however, in the first MINOR/MAJOR release of Airflow released after October 14th 2024,
```

Comment:
```suggestion
   are Python 3.8 images, however, in the first MINOR/MAJOR release of Airflow released after 2024-10-14,
```

(because spelling is complaining about `th` and this is an easier fix from GH review.)